app/views/characters/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="characters-header">
+	<h1><%= image_tag "http://vignette3.wikia.nocookie.net/starwars/images/5/52/Force_and_Destiny_logo.png/revision/latest?cb=20151102133120" %><span class="space_between_image">CHARACTERS</span></h1>
+</div>
+
+<div class="row characters-row">
+	<% @characters.each do |character| %>
+		<div class="col-sm-4 characters-col">
+			<div class="card" style="width: 20rem;">
```

Comment:
Avoid inline styling(`style="width: 20rem;"`) make a class or modify the `card` class